### PR TITLE
Fixed - Hover out of State table issue

### PR DESCRIPTION
**Pull request by dipali2098**
**Description of PR**
While hovering out of the State table, now shows overall map instead of previously hovered state.
**Type of PR**

- [-] Bugfix
- [ ] New feature

**Relevant Issues**  
Fixes #...

**Checklist**

- [-] Compiles and passes lint tests
- [-] Properly formatted
- [-] Tested on desktop
- [-] Tested on phone

**Screenshots**

### CODE REVIEW

File: src/components/mapexplorer.js [hunk 1]
```diff
@@ -80,9 +80,16 @@ function MapExplorer({
         const region = getRegionFromState(
           states.find((state) => name === state.state)
         );
-        setCurrentHoveredRegion(region);
-        setPanelRegion(region);
-        onMapHighlightChange(region);
+        if (region) {
+          setCurrentHoveredRegion(region);
```
Comment (vishwac09) [inline src/components/mapexplorer.js:84]: avoid duplicate function calls. **setCurrentHoveredRegion**, **setPanelRegion**, **onMapHighlightChange** are being referred twice with just a change in param region and noregion.
Write them once and manipulate the value passed to them as.
`
let region ;
if (!region){
   region = noregion;
}
setCurrentHoveredRegion(region);
setPanelRegion(region);
onMapHighlightChange(region)
`

File: src/components/home.js [hunk 2]
```diff
@@ -19,6 +19,13 @@ import Updates from './updates';
 import Search from './search';
 import Footer from './footer';
 
+// const getRegionFromState = (state) => {
```
Comment (vishwac09) [inline src/components/home.js:22]: remove commented

**Review by vishwac09 — CHANGES_REQUESTED** [review]
Check some comments

**Comment by jeremyphilemon** [comment]
Hey, thanks for the PR! 

Please raise PRs against issues you've been assigned to and have them referenced in the PR :)

**Pull request closed without merging**